sentry/src/routes/channel.rs

```diff
@@ -392,6 +398,95 @@ pub async fn get_accounting_for_channel<C: Locked + 'static>(
     Ok(success_response(serde_json::to_string(&res)?))
 }
 
+pub async fn channel_payout<C: Locked + 'static>(
+    req: Request<Body>,
+    app: &Application<C>,
+) -> Result<Response<Body>, ResponseError> {
+    let channel_context = req
+        .extensions()
+        .get::<ChainOf<Channel>>()
+        .expect("Request should have Channel & Chain/TokenInfo")
+        .to_owned();
+
+    let body = hyper::body::to_bytes(req.into_body()).await?;
+    let to_pay = serde_json::from_slice::<ChannelPayRequest>(&body)
+        .map_err(|e| ResponseError::FailedValidation(e.to_string()))?;
+
+    let channel_campaigns = fetch_campaign_ids_for_channel(
+        &app.pool,
+        channel_context.context.id(),
+        app.config.campaigns_find_limit,
+    )
+    .await?;
+
+    let campaigns_remaining_sum = app
+        .campaign_remaining
+        .get_multiple(channel_campaigns.as_slice())
+        .await?
+        .iter()
+        .sum::<Option<UnifiedNum>>()
+        .ok_or_else(|| {
+            ResponseError::BadRequest("Couldn't sum remaining amount for all campaigns".to_string())
+        })?;
+
+    // A campaign is closed when its remaining == 0
+    // therefore for all campaigns for a channel to be closed their total remaining sum should be 0
+    if campaigns_remaining_sum.ge(&UnifiedNum::from_u64(0)) {
```

Comment:
please just use the `>`, this check is wrong as you use Greater or Equal. It can't be Equal to 0, because all request will fail.